consensus/polybft/contracts_initializer.go

```diff
@@ -0,0 +1,79 @@
+package polybft
+
+import (
+	"encoding/hex"
+	"math/big"
+
+	bls "github.com/0xPolygon/polygon-edge/consensus/polybft/signer"
+	"github.com/0xPolygon/polygon-edge/contracts"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/umbracle/ethgo/abi"
+)
+
+var (
+	initCallStaking, _ = abi.NewMethod("function initialize(" +
```

Comment:
Could you add err check there?

Reply:
This can throw error only if we don't specify method signature correctly (so it can't in our case). Maybe we should add some linter flag or something?

Reply:
No need for adding linter ignore, because we deliberately ignored error handling here. As Igor explained, this would fail only in case bug in the code: either function signature is not correct or there is some bug in ethgo library and we should fail fast on such occasions as there is no point in handling error in this case.